lammps-tutorials.tex

```diff
@@ -1146,6 +1191,69 @@ \subsubsection{Improving the script}
 \label{fig:mixing}
 \end{figure}
 
+There are a few more differences from the previous simulation.  First,
+the \textit{velocity create} command assigns an initial velocity to each
+atom.  The initial velocity is chosen so that the average initial
+temperature is equal to 1 temperature unit.  The additional keywords
+ensure that no linear momentum (\textit{mom yes}) and no angular
+momentum (\textit{rot yes}) are given to the system and that the
+generated velocities are distributed as a Gaussian.  Another improvement
+is the \textit{zero yes} keyword in the Langevin thermostat, which
+ensures that the total random force applied to the atoms is equal to
+zero.
+
+The steps to have no initial linear momentum and no net random force are
+important to avoid that the system will start to drift or move as a
+whole.  For a bulk system with periodic boundary conditions, it is
+expected to stay in place and thus when computing the temperature from
+the kinetic energy, we use $3N-3$ degrees of freedom since there is no
+global translation.  In a drifting system, some of the kinetic energy is
+due to the drift, which means the system itself becomes cooler.  In the
+extreme case it can freeze and drift very fast and thus people sometimes
+refer to this as the ``flying ice cube syndrome''.
```

Comment:
```suggestion
due to the drift, which means the system itself cools down.  In the
extreme case it can freeze and drift very fast. This phenomenon is sometimes referred to as the ``flying ice cube syndrome'' \cite{wong2016good}.
```